src/hooks/useProfileModalFormDefaultValues.ts

```diff
@@ -24,7 +24,7 @@ export const useProfileModalFormDefaultValues: (
   useMemo(
     () => ({
       profileLinks: user.profileLinks ?? [],
-      pictureUrl: determineAvatar({ user }),
+      pictureUrl: determineAvatar({ user })[0],
```

Comment:
Switching from array to object as the result will make this easier to understand:

```suggestion
      pictureUrl: determineAvatar({ user }).src,
```

Reply:
done: https://github.com/sparkletown/sparkle/pull/2760/commits/9cc6e165b25244d4599206263d7e1e47ab5f583f